---
int32 n_objects
string object_list
